code/game/objects/structures/depot.dm

```diff
@@ -113,6 +113,8 @@
 		depotarea.updateicon()
 
 	for(var/obj/structure/closet/L in range(30, T))
+		for(var/obj/O in L)
+			qdel(O)
 		L.open()
 	for(var/mob/living/M in range(30, T))
 		M.gib()
```

Comment:
calling `Destroy` directly is something you should *never* do under any circumstances ****ever***. 

This hsould probably just deal a crap ton of damage to them to break them.

Reply:
Changed to take_damage, verified works, resolving.